tests/unit/moduleapi/aclcheck.tcl

```diff
@@ -63,13 +63,16 @@ start_server {tags {"modules acl"}} {
         # rm call check for key permission (y: only WRITE)
         assert_equal [r aclcheck.rm_call set y 5] OK
         assert_error {*NOPERM*} {r aclcheck.rm_call set y 5 get}
-        assert_error {ERR acl verification failed, can't access at least one of the keys mentioned in the command arguments.} {r aclcheck.rm_call_with_errors set y 5 get}
+        catch {r aclcheck.rm_call_with_errors set y 5 get} e
```

Comment:
any reason not to keep using assert_error? it does support patterns...

Reply:
think I was just mirroring other places, but changed them all to assert_error in this test.  no more catch/match